src/main/java/org/repodriller/RepositoryMining.java

```diff
@@ -42,18 +40,19 @@
 
 public class RepositoryMining {
 
-	private List<SCMRepository> repos;
-	private Map<CommitVisitor, PersistenceMechanism> visitors;
-	
+	private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy HH:mm:ss");
```

Comment:
should we move this to a Util class, like `DateUtils` and provide a static method to do the formatting?